osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/services/SecretManagementService.java

```diff
@@ -0,0 +1,158 @@
+package org.opensmartgridplatform.adapter.protocol.dlms.application.services;
+
+import org.apache.commons.codec.binary.Hex;
+import org.apache.commons.lang3.NotImplementedException;
+import org.opensmartgridplatform.adapter.protocol.dlms.application.wsclient.SecretManagementClient;
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.entities.DlmsDevice;
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.entities.SecurityKeyType;
+import org.opensmartgridplatform.adapter.protocol.dlms.exceptions.ProtocolAdapterException;
+import org.opensmartgridplatform.schemas.security.secretmanagement._2020._05.GetSecretsRequest;
+import org.opensmartgridplatform.schemas.security.secretmanagement._2020._05.GetSecretsResponse;
+import org.opensmartgridplatform.schemas.security.secretmanagement._2020._05.SecretType;
+import org.opensmartgridplatform.schemas.security.secretmanagement._2020._05.SecretTypes;
+import org.opensmartgridplatform.schemas.security.secretmanagement._2020._05.TypedSecret;
+import org.opensmartgridplatform.schemas.security.secretmanagement._2020._05.TypedSecrets;
+import org.opensmartgridplatform.shared.exceptionhandling.FunctionalException;
+import org.opensmartgridplatform.shared.security.EncryptionService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.beans.factory.annotation.Qualifier'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)